lib/Core/Executor-options.cpp

```diff
@@ -0,0 +1,4013 @@
+//===-- Executor.cpp ------------------------------------------------------===//
```

Comment:
This seems like an accidentally committed copy of `Executor.cpp`.

Reply:
@jbuening Oops, good catch, thanks!  I fixed this.